packaging/dependencies.nix

```diff
@@ -194,6 +208,7 @@ scope:
     bsdNoLinkAsNeeded
     localSourceLayer
     mesonLayer
+    setVersionLayer
```

Comment:
Doesn't the inclusion of `setVersionLayer` in `mkMesonLibrary` mean that every commit triggers a rebuild of all libraries?